doc/articles/migrating-from-previous-releases.md

```diff
@@ -2,15 +2,15 @@
 
 This article details the migration steps required to migrate from one version to the next.
 
-### Uno Platform 4.5
+### Uno 4.5
```

Comment:
This is the right name :)

```suggestion
### Uno Platform 4.5
```

Reply:
Should the other Uno Heading be changed to Uno Platform in the doc?